### DISCUSSION TITLE
not serving over https

### DISCUSSION THREAD

**Discussion by jojolapin972** [nginx-proxy/nginx-proxy#1947] [Q&A]
I have a wildcard certificate for my domain that I saved as /etc/nginx/certs/default.key and /etc/nginx/certs/default.crt. So I assumed that they would be picked up to serve HTTPS traffic.
But when I reach my virtual host over https I get a "503 Service Temporarily Unavailable" page.

**Comment by LM1LC3N7** [comment]
I had the same problem. It was with Gitlab if I remember right. So I have disabled the HTTPS option of GitLab, and instead I have used "nginx-proxy-companion" to generate https certificats "on the air" !
